BankChain/app/src/androidTest/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTest.java

```diff
@@ -144,4 +153,100 @@ public void testExceptionConfirmationNotFuture() {
 
         assertTrue(bank.confirmException(thr) == thr);
     }
+
+    @Test
+    public void testGetUsers() throws ExecutionException, InterruptedException {
+        CompletableFuture<Void> future;
+        Bank bank;
+        Environment env;
+
+        env = new Environment();
+        env.setBank("Bunq");
+
+        env.setBankUrl("REAL_BUNQ_URL");
+        env.setBankApiKey("REAL_BUNQ_API_KEY");
+
+        List<Party> users;
+        bank = new BankFactory(env).create();
+        bank.createSession().get();
+        users = bank.listUsers().get();
+        Party firstUser = users.get(0);
+        assertEquals(firstUser.getId(), 2002);
+        assertEquals(firstUser.getName(), "Overton Onderlinge Waarborgmaatschappij");
+    }
+
+
+    @Test
+    public void testGetAccounts() throws ExecutionException, InterruptedException {
```

Comment:
Please test that multiple accounts exist, since it is a list. Also add a test for empty list etc..